FortranCon 2021 stdlib abstract (#10)
fortran-lang/talks: FortranCon2021-stdlib/README.md

> # FortranCon2020-stdlib

Reviewer: 2021?

Reviewer 2: ```suggestion
# FortranCon2021-stdlib
```

**Authors**:
[Nathaniel Shaffer](https://github.com/nshaffer) (University of Rochester), 
[Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champagne), 
[Ondřej Čertik](https://github.com/certik) (Los Alamos National Laboratory),
[William Clodius](https://githiub.com/wclodius2),

> [Robert Cohn](https://github.com/rscohn2) (Intel Corporation)

Reviewer 3: ```suggestion
```

I only contributed a minor fix to a CI file. You can omit me. Thanks.

[Milan Curcic](https://github.com/milancuric) (University of Miami),
[Laurence Kedward](https://github.com/lkedward) (University of Bristol),
[Sebastian Ehlert](https://github.com/awvwgk) (University of Bonn),

> [gareth-nx ???](https://github.com/gareth-nx),

Reviewer 4: [gareth-nx Gareth Davies](https://github.com/gareth-nx), 

For administrative reasons it's simpler for me to have no affiliation (also this is unrelated to my day job).

Author: ```suggestion
[Gareth Davies](https://github.com/gareth-nx),
```

[Aman Godara](https://github.com/Aman-Godara),
[Jing ???](https://github.com/Jim-215-Fisher),
[Chetan Karwa](https://github.com/chetankarwa),
[Arjen Markus](https://github.com/arjenmarkus),

> [Ian Giestas Pauli](https://github.com/14NGiestas) (São Paolo University),

Reviewer 5: ```suggestion
[Ian Giestas Pauli](https://github.com/14NGiestas) (University of São Paulo),
```
I also made just a quick CI fix, so feel free to omit it. I've tried to contribute with discussions and some code review early on but not so much.

[Wileam Phan](https://github.com/wyphan) (University of Tennessee, Knoxville), 
[Ivan Pribec](https://github.com/ivan-pi), (Technical University of Munich),
[Yuichiro Sakamoto](https://github.com/sakamoti),
[Harris Snyder](https://github.com/hsnyder), 

> [Jeremie Vandenplas](https://github.com/jvdp1) (Wageningen University & Research),

Reviewer 6: Thank you for starting this. 
Good for me. I'll come back on the abstract ASAP.

[Evan Voyles](https://github.com/ejovo13) (Kalamazoo College),
[Carlos Une](https://github.com/brocolis)

**To be acknowledged in the presentation (following comments):**

**Track**: Junior presentations (15 + 5 minutes discussion)

**Presenter**: [Nathaniel Shaffer](https://github.com/nshaffer)

> **Title**: Fortran Standard Library

Reviewer 7: What do you think about making title a bit more specific, for example:

```suggestion
**Title**: What's new in the Fortran Standard Library?
```

or

```suggestion
**Title**: Second year of the Fortran Standard Library
```

Author: I like that "What's new..." title. Thanks for the suggestion.

**Abstract**:

> The Fortran standard library is now well into its second year of development.

Reviewer 6: ```suggestion
The Fortran standard library (stdlib) is now well into its second year of development.
```

> In this past year, stdlib has grown not only in its functionality but in its contributor base, with most of the new development coming from new contributors, including two Google Summer of Code students. 

Reviewer 7: What do you think about adding a sentence here about new functionality?

```suggestion
In this past year, stdlib has grown not only in its functionality but in its contributor base, with most of the new development coming from new contributors, including two Google Summer of Code students.
Stdlib provides new functionality for working with bit sets, logging, quadrature, sorting, strings, and more.
```

> In addition to new capabilities in stdlib itself, there have also been several technical improvements to the documentation, workflow, and build system.

Reviewer 6: ```suggestion
For example, in addition to new capabilities in stdlib itself, there have also been several technical improvements to the documentation, and build system.
The full API documentation is generated by the FORD tool, and can be found online.
Furthermore, the Github 's Continuous Integration pipeline is used to test all procedures derived types and modules at all pull requests, to ensure cross-platform support.
```

Author: Thanks, Jeremie. I'll adjust the phrasing to my taste later, but these are nice suggestions.

# FortranCon2021-stdlib

**Authors**:
[Nathaniel Shaffer](https://github.com/nshaffer) (University of Rochester), 
[Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champagne), 
[Ondřej Čertik](https://github.com/certik) (Los Alamos National Laboratory),
[William Clodius](https://githiub.com/wclodius2),
[Robert Cohn](https://github.com/rscohn2) (Intel Corporation)
[Milan Curcic](https://github.com/milancuric) (University of Miami),
[Laurence Kedward](https://github.com/lkedward) (University of Bristol),
[Sebastian Ehlert](https://github.com/awvwgk) (University of Bonn),
[gareth-nx ???](https://github.com/gareth-nx),
[Aman Godara](https://github.com/Aman-Godara),
[Jing ???](https://github.com/Jim-215-Fisher),
[Chetan Karwa](https://github.com/chetankarwa),
[Arjen Markus](https://github.com/arjenmarkus),
[Ian Giestas Pauli](https://github.com/14NGiestas) (São Paolo University),

> [Wileam Phan](https://github.com/wyphan) (University of Tennessee, Knoxville), 

Reviewer 8: I only contributed a minor fix for CMake, so please feel free to omit my name, for now. Looking forward to contribute more in the future, though!
```suggestion
```

# FortranCon2021-stdlib

**Authors**:
[Nathaniel Shaffer](https://github.com/nshaffer) (University of Rochester), 

> [Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champagne), 

Reviewer 9: ```suggestion
[Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champaign), 
```
I am also opting in. Thanks!

[Ondřej Čertik](https://github.com/certik) (Los Alamos National Laboratory),
[William Clodius](https://githiub.com/wclodius2),
[Robert Cohn](https://github.com/rscohn2) (Intel Corporation)

> [Milan Curcic](https://github.com/milancuric) (University of Miami),

Reviewer: ```suggestion
[Milan Curcic](https://github.com/milancurcic) (University of Miami),
```
@milancurcic

Author: Good catch!

[Laurence Kedward](https://github.com/lkedward) (University of Bristol),
[Sebastian Ehlert](https://github.com/awvwgk) (University of Bonn),
[Gareth Davies](https://github.com/gareth-nx),
[Aman Godara](https://github.com/Aman-Godara),
[Jing ???](https://github.com/Jim-215-Fisher),
[Chetan Karwa](https://github.com/chetankarwa),
[Arjen Markus](https://github.com/arjenmarkus),
[Ian Giestas Pauli](https://github.com/14NGiestas) (São Paolo University),
[Wileam Phan](https://github.com/wyphan) (University of Tennessee, Knoxville), 
[Ivan Pribec](https://github.com/ivan-pi), (Technical University of Munich),

> [Yuichiro Sakamoto](https://github.com/sakamoti),

Reviewer 10: I only modified some code to work with a particular compiler, so feel free to omit me. 
Thank you so much.
```suggestion
```

[Harris Snyder](https://github.com/hsnyder), 
[Jeremie Vandenplas](https://github.com/jvdp1) (Wageningen University & Research),
[Evan Voyles](https://github.com/ejovo13) (Kalamazoo College),

> [Carlos Une](https://github.com/brocolis)

Reviewer 11: ```suggestion
```
I opt-out. Thanks.

**To be acknowledged in the presentation (following comments):**

**Track**: Junior presentations (15 + 5 minutes discussion)

**Presenter**: [Nathaniel Shaffer](https://github.com/nshaffer)

**Title**: What's new in the Fortran Standard Library?

**Abstract**:
The Fortran standard library (stdlib) is now well into its second year of development.
In this past year, stdlib has grown not only in its functionality but in its contributor base, with most of the new development coming from new contributors, including two Google Summer of Code students.

> Stdlib provides new functionality for working with bit sets, logging, quadrature, sorting, strings, and more.

Reviewer 2: ```suggestion
Stdlib provides new functionality for working with bitsets, logging, quadrature, sorting, strings, and more.
```

> For example, in addition to new capabilities in stdlib itself, there have also been several technical improvements to the documentation, and build system.

Reviewer 12: ```suggestion
For example, in addition to new capabilities in stdlib itself, there have also seen several technical improvements to the documentation, and build system.
```

Reviewer 12: My bad, the original is correct.

The full API documentation is generated by the FORD tool, and can be found online.

> Furthermore, the Github 's Continuous Integration pipeline is used to test all procedures derived types and modules at all pull requests, to ensure cross-platform support.

Reviewer 12: ```suggestion
Furthermore, the Github's Continuous Integration pipeline is used to test all procedures derived types and modules at all pull requests, to ensure cross-platform support.
```
